tutorials/appengine-python-api-firebase-auth/app.yaml

```diff
@@ -0,0 +1,10 @@
+runtime: python27
+api_version: 1
+threadsafe: true
+
+handlers:
+  - url: /.*
+    script: api.app
+
+env_variables:
+  GCLOUD_PROJECT: 'clear-beacon-145712'
```

Comment:
As long as I know that variable is not set using the dev_appserver script in development mode and the firebase-admin library needs it.